stable/postgresql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: postgresql
-version: 0.18.0
+version: 0.18.1
```

Comment:
would say to bump to `1.0.0` since this is a chart in stable folder and we had a call in the charts meeting this week to move all stable charts that are using version 0.x.x to be at least 1.0.0 for more info you can check here: https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#heading=h.57pbxthvt6k3

Reply:
I've updated chart version to `1.0.0`. While on the subject, I've also updated PostgreSQL version to `10.5`.

Reply:
please update the README.md as well